draft-ietf-quic-transport.md

```diff
@@ -1224,11 +1227,11 @@ be unable to continue using the connection IDs with the active connection.
 
 An endpoint SHOULD limit the number of connection IDs it has retired locally and
 have not yet been acknowledged. An endpoint SHOULD allow for sending and
```

Comment:
```suggestion
An endpoint SHOULD limit the number of connection IDs it has retired through
RETIRE_CONNECTION_ID frames that remain unacknowledged. An endpoint SHOULD allow for sending and
```

Reply:
This is a semantic change, but it aligns the text with its spirit.

Reply:
Yeah, this one makes me nervous.  I get that the original is a shade awkward, but I really don't want to touch this one.  More so when I see the mistake in your suggestion below.

Reply:
Yup, I understand the hesitation. I'd like to see if we can make it clearer, since at the moment, the sentence structure is going to require disambiguating (and we are going to receive requests from translators asking us to disambiguate later).